doc/holochain_101/src/zome/capabilities.md

```diff
@@ -0,0 +1,11 @@
+# Capabilities
+
+Holochain uses a modified version of the [capabilities](https://en.wikipedia.org/wiki/Capability-based_security) security model. Instead of the more familiar access control list (ACL) security model, Holochain DNA instances grant revokable capability tokens which can then be given out as a credential to be provided back when access to some function is desired. 
+
+These capability grants are recorded as private entries on an agent's chain, and are checked for every access.  We currently have implemented three different levels of capability grants:  `Public`, `Transferable`, and `Assigned`.  
```

Comment:
I'm a little lost. How do I use Capabilities as a developer now? Or that's not really a thing, all that comes in Capabilites 3?

Reply:
updated